config/cesm/machines/config_pio.xml

```diff
@@ -45,6 +45,8 @@
     <values>
       <value>pnetcdf</value>
       <value mpilib="mpi-serial">netcdf</value>
+      <value mach="lawrencium-lr3">netcdf</value>
```

Comment:
This will only use the serial netcdf interface even though you are linking a netcdf-parallel library above.

Reply:
I think forcing the serial build was preventing some very murky run-time failures associated with CLM/ELM-FATES (fates has some gigantic array spaces because we have >1000 cohort memory spaces per column). I will run a test and drop this.